common/logisticspipes/gui/GuiPowerJunction.java

```diff
@@ -52,8 +52,7 @@ protected void drawGuiContainerBackgroundLayer(float var1, int var2, int var3) {
 		mc.fontRenderer.drawString(StringUtils.translate(GuiPowerJunction.PREFIX + "StoredEnergy") + ":", guiLeft + 40, guiTop + 23, 0x404040);
 		mc.fontRenderer.drawString(StringUtils.getStringWithSpacesFromInteger(junction.getPowerLevel()) + " LP", guiLeft + 40, guiTop + 33, 0x404040);
 		mc.fontRenderer.drawString("/ " + StringUtils.getStringWithSpacesFromInteger(LogisticsPowerJunctionTileEntity.MAX_STORAGE) + " LP", guiLeft + 40, guiTop + 43, 0x404040);
-		mc.fontRenderer.drawString("10 RF = 5 LP", guiLeft + 24, guiTop + 58, 0x404040);
-		mc.fontRenderer.drawString("1 EU = 2 LP", guiLeft + 100, guiTop + 58, 0x404040);
+		mc.fontRenderer.drawString("2 MJ = 5 EU = 20 RF = 10 LP", guiLeft + 24, guiTop + 58, 0x404040);
```

Comment:
Why change it up? I don't like it like that, because for me that's not really readable.

Reply:
There wasn't really space left in the GUI, I guess I could make it bigger but that would involve a lot more effort. Or move the text closer together, I guess I could try that

Reply:
Well IIRC there used to be text on there for MJ as well. And the GUI wasn't resized since. Need to check the commit history.

Reply:
Ah, found it. That works, I guess